templates/sag-cc-repos/README.md

```diff
@@ -18,10 +18,12 @@
 -->
 # Command Central Public Product and Fix Master Repositories
 
-Use this template to create public Software AG master repositories for products and fixes in Command Central with default contents.
+Use this template to add Software AG product and fix repositories for all officially supported releases.
 
 ## Requirements
 
+None.
```

Comment:
There is a requirement: the credentials should have been created:
We need to add either the command to apply sag-cc-creds like:
sagcc exec templates composite apply sag-cc-creds \
  credentials.username=yourusername \
  credentials.password=yourpassword \
  credentials.key=EMPOWER \
  --sync-job --wait 360

Or reference to sag-cc-creds readme

Reply:
Thanks, I added the requirement.